static/app/components/events/autofix/autofixDiff.tsx

```diff
@@ -522,11 +521,7 @@ function DiffHunkContent({
             data-overlay="true"
           >
             <OverlayHeader>
-              <OverlayTitle
-                dangerouslySetInnerHTML={{
-                  __html: singleLineRenderer(t('Editing `%s`', fileName)),
-                }}
-              />
```

Comment:
why remove the code backticks around the file name?

Reply:
ohhhhh didn't realize that's what it was doing